### PR TITLE
corepack-install

### PR DESCRIPTION
**Pull request by jarrodcolburn**
- **update devcontainer to run `corepack install`**
I wasn't running 'corepack install' before when spinning up container. I think I'm supposed to.

### CODE REVIEW

**Review by sfshaza2 — APPROVED** [review]
This sounds goo to me. Can you confirm, @parlough?

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		"PATH": "${containerEnv:PATH}:${containerEnv:FLUTTER_ROOT}/bin"
 	},
 	"onCreateCommand": "git clone --branch=stable https://github.com/flutter/flutter.git ${containerEnv:FLUTTER_ROOT} && ${containerEnv:FLUTTER_ROOT}/bin/flutter doctor",
-	"postCreateCommand": "git submodule update --init --recursive && flutter pub get && pnpm install",
+	"postCreateCommand": "git submodule update --init --recursive && flutter pub get && corepack install && pnpm install",
```
Comment (domesticmouse) [inline .devcontainer/devcontainer.json:16]: Going from https://www.npmjs.com/package/corepack I'm guessing this is meant to be `corepack enable`

Comment (sfshaza2) [reply .devcontainer/devcontainer.json:16]: Is that true, @jarrodcolburn? Should it be `corepack enable`, instead of `install`?

Comment (jarrodcolburn) [reply .devcontainer/devcontainer.json:16]: Here's my belief...
* `corepack` is already enabled in the container 
* `corepack install` needs to run in the project's root

**Review by domesticmouse — APPROVED** [review]
LGTM

**Pull request merged**